src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,68 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(readFromFile(fromFileName), toFileName);
+    }
+
+    private Report readFromFile(String fileName) {
+        Report report = new Report();
+        int sumBuy = 0;
+        int sumSupply = 0;
+        int result;
+        String line;
```

Comment:
let's move this `line` declaration inside the `try` block

Reply:
moved line to `try ` block